.github/workflows/binary-releases.yml

```diff
@@ -103,7 +105,7 @@ jobs:
           wget https://www.mpfr.org/mpfr-current/mpfr-4.1.0.tar.xz && tar xaf mpfr-4.1.0.tar.xz
           ( cd mpfr-4.1.0; CC=musl-gcc ./configure --prefix /tmp/gmp-prefix -with-gmp=/tmp/gmp-prefix && make && make install )
           eval $(opam env)
-          MPFR_PREFIX=/tmp/gmp-prefix GMP_PREFIX=/tmp/gmp-prefix opam switch create . 4.11.2+musl+static+flambda --deps-only
+          MPFR_PREFIX=/tmp/gmp-prefix GMP_PREFIX=/tmp/gmp-prefix make deps-without-ocaml
```

Comment:
I assume that this line is enogu for `conf-gmp` to point to the modified GMP and MPFR version ? If yes then all good :)